pages/tasks.js

```diff
@@ -0,0 +1,29 @@
+const tasks = [
```

Comment:
Since this is not a page but some dummy data, I feel that this file should go into a different folder, perhaps `dummyData/tasks.js`?